test/unit/volatile-file.test.js

```diff
@@ -17,9 +17,7 @@ describe('VolatileFile', () => {
     writeStreamMock = jest.fn(() => writeStreamInstanceMock);
 
     file = new VolatileFile({
-      size: 1024,
       name: 'cat.png',
```

Comment:
we removed `name`. and I get that it's for demo that everything gets merged in `this`, but lets call that `xname` or something, not to confuse it that it's an actual thing.